client/my-sites/site-settings/press-this/link.jsx

```diff
@@ -23,7 +23,11 @@ const pressThis = function ( postURL ) {
 	} else if ( docGetSel ) {
 		sel = docGetSel();
 	} else {
-		sel = docSel ? docSel.createRange().text : 0;
+		sel = docSel ? docSel.createRange().text : '';
+	}
+
+	if ( ! encodeURIComponent( sel ) ) {
+		sel = doc.title;
```

Comment:
Use title as the content of quote to improve the handling of situations where no text is selected where no text is selected. This script will be used in bookmarks, so we cannot add comment.